src/threadAccessController.js

```diff
@@ -126,10 +125,16 @@ class ThreadAccessController extends EventEmitter{
 
   async grant (capability, id) {
     if (!this._db.access.isValidCapability(capability)) {
-      throw new Error('Invalid capability to grant')
+      throw new Error('grant: Invalid capability to grant')
     }
     if (!isValid3ID(id)) {
```

Comment:
Remind me why we need to check this? Seems like it would stop other orbitdb identity-providers to use this AC?

Reply:
do you mean others who want to use other capabilities besides MODERATOR or MEMBER ? its not too generalizable right now, since logic in canAppend is very specific to particular capabilities defined for the AC

Reply:
Oh, sorry I meant `isValid3ID`

Reply:
ah yeah the comments was on that line :) hmm, yeah not required, just want to sanitize inputs in someway, just so we expect what is there, but there is no reason this could not move out of AC and just into 3box code instead